src/tasks/main.ts

```diff
@@ -150,14 +151,32 @@ export class MainAction {
           // and the wrapper scripts get updated if a new version is available (happens infrequently).
           // https://docs.gradle.org/current/userguide/gradle_wrapper.html#sec:upgrading_wrapper
           core.startGroup('Updating Wrapper (2nd update)');
-          await updater.update();
+          try {
+            await updater.update();
+          } catch (ex) {
+            if (this.inputs.ignoreFailureAfterUpdate) {
+              buildBrokenAfterBump = true;
+              core.warning(ex instanceof Error ? ex : `${ex}`);
+              core.warning('Ignoring failure after 2nd update');
+            } else {
+              throw ex;
+            }
+          }
           modifiedFiles = await git.gitDiffNameOnly();
           core.debug(`Modified files count: ${modifiedFiles.length}`);
           core.debug(`Modified files list: ${modifiedFiles}`);
           core.endGroup();
 
           core.startGroup('Verifying Wrapper');
-          await updater.verify();
+          try {
+            await updater.verify();
+          } catch (ex) {
+            if (this.inputs.ignoreFailureAfterUpdate) {
+              core.warning(ex instanceof Error ? ex : `${ex}`);
+            } else {
+              throw ex;
+            }
+          }
```

Comment:
Do we want to ignore exceptions here as well? I think that verification should pass even if there's been an error with the second `update()` call, right?

Reply:
That's correct. I'll fix it.

Don't remember why I added it now, I may have assumed that `./gradlew --help` would fail if the build is broken, but it doesn't